examples/websocket-relay/relay-app/src/components/node/Node.svelte

```diff
@@ -0,0 +1,123 @@
+  export let bgColor: CSSColorString = "white";
+  export let borderColor: CSSColorString = "transparent";
+  export let dimensions = {
+    height: 150,
+    width: 150,
+  };
+  export let label = "Default Node";
+  export let tempcat = false;
+  export let id: string;
+  export let position: Position;
+  export let task: Task | null = null;
+
+  const matchingEdge = $edgeStore.find((edge) => edge?.target === id)
+</script>
+
+<Node
+  {id}
+  {...dimensions}
+  {bgColor}
+  {borderColor}
+  {label}
+  {position}
+>
+  <div class="relative w-full h-full">
+    {#if task === null}
+      {#if !tempcat}
+        <img
+          src={`data:image/png;base64,${base64Cat}`}
+          draggable="false"
+          alt="A cat in space chilling on a synth."
+        />
+        <Anchor
+          id="1-east"
+          multiple
+          invisible
+          direction="east"
+          dynamic
+        />
+      {:else}
+        <img
+          src={`data:image/png;base64,${base64Cat}`}
+          draggable="false"
+          alt="A cat in space chilling on a synth."
+        />
+        <Anchor
+          id={`${id}-west`}
+          multiple
+          invisible
+          direction="west"
+          connections={[["1", "1-east"]]}
+          dynamic
+        >
+          <Edge slot="edge" />
+        </Anchor>
+      {/if}
+    {:else if task.status === "replayed"}
+      <img
+        src={`data:image/png;base64,${task.receipt?.out[1]}`}
+        draggable="false"
+        style="filter: opacity(75%)"
+        alt={`A cat image after a ${task.operation} performed by Homestar. The operation was replayed.`}
+      />
+      {#if matchingEdge}
+        <!-- If difference between `target` and `source` is greater than 1, we're breaking to a new row, so we'll use north/south directions -->
+        {#if (matchingEdge.target - matchingEdge.source) > 1}
+          <Anchor
+            id={`${id}-north`}
+            multiple
+            invisible
+            direction="north"
+            connections={[[`${matchingEdge.source}`, `${matchingEdge.source}-south`]]}
+            edgeLabel={matchingEdge.label}
+            dynamic
+          >
+            <Edge slot="edge" />
+          </Anchor>
+        {:else}
+          <Anchor
+            id={`${id}-west`}
+            multiple
+            invisible
+            direction="west"
+            connections={[[`${matchingEdge.source}`, `${matchingEdge.source}-east`]]}
+            edgeLabel={matchingEdge.label}
+            dynamic
+          >
+            <Edge slot="edge" />
+          </Anchor>
+        {/if}
+      {/if}
+    {:else}
+      <img
+        src={`data:image/png;base64,${task.receipt?.out[1]}`}
+        draggable="false"
+        alt={`A cat image after a ${task.operation} performed by Homestar`}
+      />
+    {/if}
+    <Anchor
+      id={`${id}-east`}
+      multiple
+      invisible
+      direction="east"
+      dynamic
+    />
+    <Anchor
```

Comment:
The conditional rendering logic in here was getting a bit extreme, so I decided to render an `east` and `south` facing Anchor in all the Nodes for when they're needed. I don't think we lose much by having them rendered here and it's useful for when Edges need to connect from those directions 🤷🏼

Reply:
Yeah totally, I mean they're invisible so totally alright to default have them. 👌